ui/src/components/inputs/InputsForm.vue

```diff
@@ -7,6 +7,23 @@
             :required="input.required !== false"
             :prop="input.id"
         >
+            <!-- Radio Button Group Rendering -->
+                        <el-radio-group
```

Comment:
All these errors need fixing @yoyounik.

Reply:
Yes thanks @MilosPaunovic it was a mistake i'll update it , thanks for this.